examples/organiq-raspberry/pi-client.js

```diff
@@ -0,0 +1,29 @@
+
+//////////////////////////////////////////////////////////////////////////////

```

Comment:
Block comments should be either:

/*
  Comment contents here
*/

Or 

// 
// this is a block comment
//